ajax/libs/hola_player/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "hola_player",
+  "description": "An HTML5 and Flash video player with a common API and skin for both.",
+  "version": "1.0.54",
+  "filename": "hola_player.js",
+  "license": "ISC",
+  "keywords": [
+    "flash",
+    "html5",
+    "player",
+    "video",
+    "videojs"
+  ],
+  "homepage": "http://holacdn.com/player",
+  "author": "Steve Heffernan",
```

Comment:
@extend1994 I'm sorry for filter in wrong author. 
The author is Bahaa Aidi.  ref: https://github.com/hola/hola_player/graphs/contributors?from=2016-12-04&to=2016-12-11&type=c
I'll modify it when I get the answer of homepage url.